cypress/integration/articleFlows/mangageFlows/lockDiscussion.spec.js

```diff
@@ -0,0 +1,107 @@
+describe('Lock discussion', () => {
+  const getDiscussionLockButton = () =>
+    cy.findByRole('link', {
+      name: /Lock discussion/,
+    });
+
+  const getDiscussionUnlockButton = () =>
+    cy.findByRole('link', {
+      name: /Unlock discussion/,
+    });
+
+  const getDiscussionLockSubmitButton = () => cy.get('.crayons-btn--danger');
+
+  const exampleReason = 'Discussion lock example reason!';
+  const exampleNotes = 'Discussion lock example notes!';
+
+  describe('locking the discussion', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.createArticle({
+            title: 'Test Article',
+            tags: ['beginner', 'ruby', 'go'],
+            content: `This is a test article's contents.`,
+            published: true,
+          }).then((response) => {
+            cy.visit(`${response.body.current_state_path}/manage`);
+          });
+        });
+      });
+    });
+
+    it('should allow a user to lock a discussion', () => {
+      getDiscussionLockButton().click();
+      getDiscussionLockSubmitButton().click();
+
+      getDiscussionUnlockButton().should('exist');
+    });
+
+    it('should ask the user to confirm the discussion lock', () => {
+      getDiscussionLockButton().click();
+
+      cy.findByRole('heading', {
+        name: 'Are you sure you want to lock the discussion on this article?',
+      }).should('exist');
+
+      cy.findByRole('button', {
+        name: /Lock discussion/,
+      }).should('exist');
+    });
```

Comment:
I don't have any strong feeling about it, but I think this _could_ be combined with the previous test